docs/migration-config/arbitrum-config.toml

```diff
@@ -0,0 +1,54 @@
+[indexer]
+indexer_address = "${INDEXER_SERVICE_INDEXER_ADDRESS}"
+operator_mnemonic = "${INDEXER_SERVICE_MNEMONIC}"
+
+[database]
+host = "${INDEXER_SERVICE_POSTGRES_HOST}"
+user = "${INDEXER_SERVICE_POSTGRES_USERNAME}"
+password = "${INDEXER_SERVICE_POSTGRES_PASSWORD}"
+database = "${INDEXER_SERVICE_POSTGRES_DATABASE}"
+
+[graph_node]
+# URL to your graph-node's query endpoint
+query_url = "${INDEXER_SERVICE_GRAPH_NODE_QUERY_ENDPOINT}"
+# URL to your graph-node's status endpoint
+status_url = "${INDEXER_SERVICE_GRAPH_NODE_STATUS_ENDPOINT}"
+
+[subgraphs.network]
+# Query URL for the Graph Network subgraph.
+query_url = "${INDEXER_SERVICE_NETWORK_SUBGRAPH_ENDPOINT}"
+deployment_id = "${INDEXER_SERVICE_NETWORK_SUBGRAPH_DEPLOYMENT}"
+
+# [subgraphs.escrow]
+# INDEXER_SUBGRAPHS__ESCROW__QUERY_URL
+# query_url = ""
+# INDEXER_SUBGRAPHS__ESCROW__DEPLOYMENT_ID
+# deployment_id = ""
```

Comment:
Why is this here like this?

Reply:
These are two missing fields that need to be provided by the user